src/components/presentational/WeekCalendar/WeekCalendar.css

```diff
@@ -1,101 +1,88 @@
-        overflow: hidden;
-        position: relative;
-        display: inline-block;
-        vertical-align: top;
-    }
+.mdhui-week-calendar .mdhui-week-calendar-day {
+    width: calc(14.28% - 1px);
+    border-left: solid 1px var(--mdhui-border-color-1);
+    overflow: hidden;
+    position: relative;
+    display: inline-block;
+    vertical-align: top;
+}
 
-        .mdhui-week-calendar .mdhui-week-calendar-day:first-child
-        {
-            width: 14.28%;
-            border-left: none;
-        }
+.mdhui-week-calendar .mdhui-week-calendar-day:first-child {
+    width: 14.28%;
+    border-left: none;
+}
 
-    .mdhui-week-calendar .mdhui-week-calendar-date-label
-    {
-        padding-top: 4px;
-        line-height: 16px;
-        padding-bottom: 8px;
-        text-align: center;
-    }
+.mdhui-week-calendar .mdhui-week-calendar-date-label {
+    padding-top: 4px;
+    line-height: 16px;
+    padding-bottom: 8px;
+    text-align: center;
+}
 
-        .mdhui-week-calendar .mdhui-week-calendar-date-label .mdhui-week-calendar-day-of-week
-        {
-            font-size: 10px;
-        }
+.mdhui-week-calendar .mdhui-week-calendar-date-label .mdhui-week-calendar-day-of-week {
+    font-size: 10px;
+}
 
-        .mdhui-week-calendar .mdhui-week-calendar-date-label .mdhui-week-calendar-day-of-month
-        {
-            font-size: 14px;
-            display: inline-block;
-            text-align: center;
-            width: 24px;
-            height: 24px;
-            border-radius: 24px;
-            line-height: 24px;
-            margin-top: 2px;
-            margin-bottom: 2px;
-        }
+.mdhui-week-calendar .mdhui-week-calendar-date-label .mdhui-week-calendar-day-of-month {
+    font-size: 14px;
+    display: inline-block;
+    text-align: center;
+    width: 24px;
+    height: 24px;
+    border-radius: 24px;
+    line-height: 24px;
+    margin-top: 2px;
+    margin-bottom: 2px;
+}
 
-    .mdhui-week-calendar .mdhui-week-calendar-day-selectable
-    {
-        cursor: pointer;
-    }
+.mdhui-week-calendar .mdhui-week-calendar-day-selectable {
+    cursor: pointer;
+}
 
-    .mdhui-week-calendar .mdhui-week-calendar-day-selected
-    {
-        background: #EAF4FB;
-    }
+.mdhui-week-calendar .mdhui-week-calendar-day-selected {
+    background: var(--mdhui-background-color-2);
+}
 
-        .mdhui-week-calendar .mdhui-week-calendar-day-selected .mdhui-week-calendar-date-label .mdhui-week-calendar-day-of-month
-        {
-            background-color: var(--color-theme);
-            color: #FFF;
-            font-weight: bold;
-        }
+.mdhui-week-calendar .mdhui-week-calendar-day-selected .mdhui-week-calendar-date-label .mdhui-week-calendar-day-of-month {
+    background-color: var(--mdhui-color-primary);
+    color: #FFF;
+    font-weight: bold;
+}
 
-    .mdhui-week-calendar .mdhui-week-calendar-day-future > *
-    {
-        opacity: .4;
-    }
+.mdhui-week-calendar .mdhui-week-calendar-day-future>* {
```

Comment:
I think this change is harmless, but just wanted to mention it in case `.class > *` functions differently than `.class>*`.